src/core/js/device.js

```diff
@@ -38,11 +38,15 @@ define(function(require) {
         return screenSize;
     }
 
+    function getScreenWidth() {
+        return isAppleDevice()
+            ? getAppleScreenWidth()
+            : window.innerWidth || $window.width()
```

Comment:
The ending semi-colon has gone astray here. Feel free to ignore if it doesn't bother anyone!